test/e2e/networkpolicy_test.go

```diff
@@ -935,6 +935,21 @@ func createAndWaitForPod(t *testing.T, data *TestData, createFunc func(name stri
 	return name, podIP, cleanupFunc
 }
 
+func createAndWaitForPodWithExactName(t *testing.T, data *TestData, createFunc func(name string, ns string, nodeName string, hostNetwork bool) error, name string, nodeName string, ns string, hostNetwork bool) (string, *PodIPs, func()) {
```

Comment:
updated. `createAndWaitForPod` is in  `networkpolicy_test.go` as well. I put `createAndWaitForPodWithExactName` into `createAndWaitForPod` func body to avoid redundant code.